classes/base.lua

```diff
@@ -341,22 +361,23 @@ function class:registerCommands ()
     end
   end, "Includes a content file for processing.")
 
-  self:registerCommand("lua", function (options, content)
-    local packopts = packOptions(options)
-    if SU.hasContent(content) then
-      local doc = SU.contentToString(content)
-      return SILE.processString(doc, "lua", nil, packopts)
-    elseif options.src then
-      return SILE.processFile(options.src, "lua", packopts)
-    elseif options.require then
-      local module = SU.required(options, "require", "lua")
-      return require(module)
-    else
-      SU.error("\\lua function requires inline content or a src file path or a require module name")
-    end
-  end, "Run Lua code. The code may be supplied either inline, using require=... for a Lua module, or using src=... for a file path")
+  self:registerCommand("lua", function(options, content)
+      local packopts = packOptions(options)
+      if SU.hasContent(content) then
+        local doc = SU.contentToString(content)
+        return SILE.processString(doc, "lua", nil, packopts)
+      elseif options.src then
+        return SILE.processFile(options.src, "lua", packopts)
+      elseif options.require then
+        local module = SU.required(options, "require", "lua")
+        return require(module)
+      else
+        SU.error("\\lua function requires inline content or a src file path or a require module name")
+      end
+    end,
+    "Run Lua code. The code may be supplied either inline, using require=... for a Lua module, or using src=... for a file path")
```

Comment:
Another big example of untouched code reformatted without relation to this PR, etc.